### PR TITLE
[zh] Add concepts folder and its index

### PR DESCRIPTION
**Pull request by windsonsea**
Add two pages to zh folder

```
content/zh/docs/concepts/_index.md
content/zh/docs/concepts/components.md
```

See [preview](https://deploy-preview-4525--opentelemetry.netlify.app/zh/docs/concepts/components/)

### CODE REVIEW

File: content/zh/docs/concepts/components.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
+---
+title: 组件
+description: 构成 OpenTelemetry 的主要组件
+aliases: [data-collection]
+weight: 20
+---
+
+OpenTelemetry 目前由以下几个主要组件构成：
```
Comment (reyang) [inline content/zh/docs/concepts/components.md:8]: Consider:

```suggestion
OpenTelemetry 项目目前由以下几个主要部分构成：
```

Reason:

组件 is normally interpreted as software components or modules (模块).

File: content/zh/docs/concepts/components.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+
+## Collector
+
+OpenTelemetry Collector 是一个与供应商无关的代理，可以接收、处理和导出遥测数据。
+它支持以多种格式接收遥测数据（例如 OTLP、Jaeger、Prometheus 以及许多商业/专有工具）并将数据发送到一个或多个后端。
+它还支持在导出之前处理和过滤遥测数据。
+
+更多信息，请参见 [Collector](/docs/collector/)。
+
+## 特定语言的 API 和 SDK 实现 {#language-specific-api--sdk-implementations}
```
Comment (reyang) [inline content/zh/docs/concepts/components.md:42]:
```suggestion
## 针对特定编程语言的 API 和 SDK 实现 {#language-specific-api--sdk-implementations}
```

File: content/zh/docs/concepts/components.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+这些 SDK 还允许你结合常见库和框架的插桩库，以便你可以将其用于应用程序中的手动插桩。
+
+更多信息，请参见[插桩操作](/docs/concepts/instrumentation/)。
+
+### 插桩库 {#instrumentation-libraries}
+
+OpenTelemetry 支持通过大量组件来为所支持的语言根据流行的库和框架生成相关遥测数据。
+例如，来自 HTTP 库的入站和出站 HTTP 请求将生成有关这些请求的数据。
+
+长期目标是流行的库能够开箱即用地进行观测，无需到拉入单独的组件中。
```
Comment (reyang) [inline content/zh/docs/concepts/components.md:54]: This line seems confusing, doesn't sound like native Chinese?

File: content/zh/docs/concepts/components.md [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+
+更多信息，请参见[插桩库](/docs/concepts/instrumentation/libraries/)。
+
+### 导出器 {#exporters}
+
+{{% docs/languages/exporters/intro %}}
+
+### 零代码插桩 {#zero-code-instrumentation}
+
+如果适用，OpenTelemetry 的特定语言实现将提供一种无需触及源代码即可对应用程序进行插桩的方法。
```
Comment (reyang) [inline content/zh/docs/concepts/components.md:64]:
```suggestion
如果适用，OpenTelemetry 的特定语言实现将提供一种无需修改源代码即可对应用程序进行插桩的方法。
```

**Review by reyang — APPROVED** [review]
LGTM with couple non-blocking suggestions.

File: content/zh/docs/concepts/components.md (hunk 3 above)
Comment (windsonsea) [reply content/zh/docs/concepts/components.md]: Thanks, updated

File: content/zh/docs/concepts/_index.md [hunk 5]
```diff
@@ -0,0 +1,10 @@
+---
+title: OpenTelemetry 概念 
+linkTitle: Concepts
```
Comment (svrnm) [inline content/zh/docs/concepts/_index.md:3]: if you also translate the linkTitle it will also be updated in the navigation

Comment (windsonsea) [reply content/zh/docs/concepts/_index.md]: ok, translated.

**Review by svrnm — APPROVED** [review]

**Comment by svrnm** [comment]
thanks @windsonsea and @reyang, great to see the Chinese documentation growing so quickly!

**Pull request merged**